eurocropsml/configs/acquisition/cfg/latvia.yaml

```diff
@@ -7,7 +7,7 @@ country_config:
   months: [1,12]
   product_type: "L1C"
   max_requested_products: 1000
-  satellite: "S2"
+  satellite: "S1"
```

Comment:
Same question here as above (and in all other configs): Was this changed for testing? I think the general default should remain S2, no?